### ISSUE TITLE
How to anchor PopupNotifier

### ISSUE THREAD

**Issue by Blatotlacka** [Tulpep/Notification-Popup-Window#21]
Hi,

is there a way how to anchor PopupNotifier to form instead of a screen?
I tried to set location to active form. But it is not anchored to form. 

Thanks for answer.

**Comment by LiorBanai** [comment]
you want it to follow the form and not the screen?

**Comment by ricardopolo** [comment]
I’m not understanding the issue. Can you elabore a bit more please?

**Comment by Blatotlacka** [comment]
> you want it to follow the form and not the screen?

Yes, thats what i want. I tried change this: Popup.Location = new Point(ActiveForm.Right - frmPopup.Size.Width - 1 .......);
Popupnotifier follows the form then. But it stays in its place when I try to move the form.
